distribution/docker/ballerina-base-image/README.md

```diff
@@ -27,11 +27,11 @@ docker run -v /tmp/testb:/ballerina/files -p 9090:9090 -it ballerinalang/balleri
 ## Building the image
 ```bash
 # bash build.sh -d <ballerina-dist> to build ballerina:latest
-cp ~/Downloads/ballerina-0.96.1-SNAPSHOT.zip .
-bash build.sh -d ballerina-0.96.1-SNAPSHOT.zip
+cp ~/Downloads/ballerina-0.961.1-SNAPSHOT.zip .
```

Comment:
This PR has maven replacer version updates. ( Docker/ Composer )Shall we remove them? Will address this separately.